lib/browser/objects-registry.js

```diff
@@ -20,6 +20,7 @@ class ObjectsRegistry {
   add(webContents, obj) {
     // Get or assign an ID to the object.
     let id = this.saveToStorage(obj);
+    this.storage[id].count++;
```

Comment:
It is correct to track the number of references, but we should not increase the number for every call of `add`. Consider the following case:

```
add(webContents1, 1) // first reference of object 1 in webContents1
add(webContents1, 1) // second reference
remove(webContents, 1) // all references of object 1 are gone in webContents1
```

So when there is webContents referencing object 1, the number of references to object 1 would be > 0.